src/loadComponents.js

```diff
@@ -18,21 +19,23 @@ function loadState(rootState) {
           'loadable-component server modules:',
           window[LOADABLE_STATE],
         )
-        throw new Error(
-          `loadable-components: module "${
-            state.id
-          }" is not found, client and server modules are not sync. You are probably not using the same resolver on server and client.`,
-        )
+        const errorMessage = `loadable-components: module "${
+          state.id
+        }" is not found, client and server modules are not sync. You are probably not using the same resolver on server and client.`
+
+        console.error(errorMessage)
+        return Promise.reject(errorMessage)
```

Comment:
I think we should just do `return Promise.reject(new Error(...))`, it should be sufficient. So :

- Please remove log
- Please call `Promise.reject` with an error

Reply:
`return Promise.reject(new Error(...))` is much cleaner 👍